app/views/contests/_form.html.erb

```diff
@@ -46,8 +53,16 @@
     <% else %>
       <%= handle(@contest.owner) %>
     <% end %>
-
   </div>
+  <div class="field">
+    <%= f.check_box :live_scoreboard %>
+    <%= f.label :live_scoreboard %>Show the scoreboard during the contest. If unchecked, users can only see their own score until the end of the contest.<br />
```

Comment:
This results in the following text:

> Live scoreboardShow the scoreboard during the contest. If unchecked, users can only see their own score until the end of the contest.

Suggest either prepending " - " to the description as before, or just removing the "Live scoreboard" part:
```suggestion
    <%= f.label :live_scoreboard, "Show the scoreboard during the contest. If unchecked, users can only see their own score until the end of the contest." %><br />
```
(same with `only_rank_official_contestants`)

Reply:
Oops, I didn't realise there was an automatic label too. I added the "-" back for now.
Replacing the automatic label ("Live scoreboard") with the long description as in your suggested edit makes the entire description clickable, which I don't like (too easy to accidentally toggle).
Maybe we should use parenthesised italics for the description, as done for the password field:
https://github.com/NZOI/nztrain/blob/84633ca11a3cac7d7d6fb9dffa885f8a90350b24/app/views/accounts/registrations/edit.html.erb#L15
I'm thinking of rewording `only_rank_official_contestants` so it displays as "Only rank official contestants - Exclude contestants from ranking if they are not enrolled in a NZ school."

Reply:
I don't mind switching to parenthesised italics, maybe with some extra spacing between the label and description.

>  I'm thinking of rewording `only_rank_official_contestants` so it displays as "Only rank official contestants - Exclude contestants from ranking if they are not enrolled in a NZ school."

Sounds good.